QUARKUS-478.md

```diff
@@ -0,0 +1,41 @@
+# QUARKUS-478 Quarkus should provide a first class experience for using native mode on OpenShift Serverless
+
+Jira: https://issues.redhat.com/browse/QUARKUS-478
+GitHub: https://github.com/quarkusio/quarkus/pull/11760
+
+The goal of QUARKUS-478 is to provide unified experience of build and deploying Quarkus applications to OpenShift. This
+shall be achieved by extending `quarkus-container-image-openshift` extension to enable user to create the build and 
+deployment of application with single, unified command for either JVM and native applications with both classic and 
+serverless application architecture.
+
+## Scope of the testing
```

Comment:
I think `quarkus.openshift.jvm-dockerfile=MyDockerFile` and native equivalent should be also covered